src/theme/docs/components/Colors.tsx

```diff
@@ -0,0 +1,125 @@
+import { Box, Text } from "~/components";
```

Comment:
**question**: Shouldn't this helper be located inside the `.storybook` folder?

Thanks to that we could avoid the accidental import of helper that can end up inside production bundle